browser/css/jsdialogs.css

```diff
@@ -568,6 +573,25 @@ td.jsdialog > [id^='table-box']:not(.sidebar) {
 	/* grid-column: '1 / ' + (this._columns + 1) - set inside JS */
 }
 
+#ServerAuditDialog .ui-treeview-headers {
+	display: grid;
+	grid-template-columns: auto 1fr 170px;
+}
+
+#ServerAuditDialog .ui-treeview-icon {
+	margin-inline-end: 2px;
+}
+
+#ServerAuditDialog .ui-treeview-headers div.ui-treeview-icon-column {
+	margin-inline-end: 4px;
+	width: 0px /* prevent display of ui-treeview-icon-column in ui-treeview-header */
+}
+
+#ServerAuditDialog .ui-treeview-entry > div:nth-last-child(1) {
+	margin-inline-start: 8px;
+}
```

Comment:
We don't need margins if we use column-gap as in:

```css
.ui-treeview-headers,
+.ui-treeview-entry {
+	column-gap: 8px;
+}
```

Reply:
Using `column-gap: 8px` on `ui-treeview-headers`  adds a gap between all three grid items, but we want to avoid adding extra space between the icon column and the status.